 src/CMakeLists.txt | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 4395d79..3899808 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -15,7 +15,7 @@ endif(NOT DEFINED CMAKE_POSITION_INDEPENDENT_CODE)
 
 # Get Qt5
 find_package(Qt5 COMPONENTS Core REQUIRED)
-find_package(HDF5 1.12.0 CONFIG REQUIRED)
+find_package(HDF5 CONFIG REQUIRED)
 
 set(CMAKE_CXX_STANDARD 14)
 
@@ -92,11 +92,8 @@ else()
     target_compile_definitions(GenH5 PUBLIC GENH5_STATIC)
 endif()
 
-if (TARGET hdf5::hdf5 AND NOT TARGET hdf5-shared)
-  add_library(hdf5-shared ALIAS hdf5::hdf5)
-endif()
 
-target_link_libraries(GenH5 PUBLIC Qt5::Core hdf5-shared)
+target_link_libraries(GenH5 PUBLIC Qt5::Core hdf5::hdf5-shared)
 
 target_include_directories(GenH5 PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/>
